src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -25,6 +25,12 @@ typedef halide_hexagon_remote_scalar_t scalar_t;
 
 extern "C" {
 
+void qdebug_dump_reg_asm();
+
+void halide_vprint(void *user_context, const char *str) {
+    qdebug_dump_reg_asm();
```

Comment:
We have to find a way to print "str". This is because Halide print can be used like so
`g(y) = f(x-1) + print ("This is the value of f(x) ->", f(x));`